networking/eth2/src/test/java/tech/pegasys/teku/networking/eth2/rpc/beaconchain/methods/BlobsSidecarsByRangeMessageHandlerTest.java

```diff
@@ -193,6 +194,27 @@ public void shouldSendToPeerRequestedNumberOfBlobsSidecars() {
     assertThat(actualSent).containsExactlyElementsOf(expectedSent);
   }
 
+  @Test
+  public void shouldIgnoreRequestWhenCountIsZero() {
```

Comment:
no need to to fix anything here I think as we are going to remove it

Reply:
My bad! Mixed up.
Removed